storagenode/pieces/store.go

```diff
@@ -32,7 +32,7 @@ type Info struct {
 
 	PieceID         storj.PieceID
 	PieceSize       int64
-	PieceExpiration time.Time
+	PieceExpiration *time.Time
```

Comment:
What's the benefit of changing this to a pointer? I don't see it checked on any place in this PR.

Reply:
First of all `PieceExpiration` is not saved to DB this way, on storage node we are receiving nil timestamp so I tough that converting this to empty `time.Time` makes no sense.

Reply:
OK.